starter-core/src/main/java/io/micronaut/starter/feature/lang/kotlin/Kotlin.java

```diff
@@ -55,7 +55,7 @@ public void processSelectedFeatures(FeatureContext featureContext) {
 
     @Override
     public void apply(GeneratorContext generatorContext) {
-        generatorContext.getBuildProperties().put("kotlinVersion", "1.4.30");
+        generatorContext.getBuildProperties().put("kotlinVersion", "1.4.32");
```

Comment:
Why is this hard coded?

Reply:
It was hardcoded before this PR. I just bump up the version. But I will extract it to the pom.xml and we can get rid of this magic number.

Reply:
I created a separate PR https://github.com/micronaut-projects/micronaut-starter/pull/747